### PR TITLE
Add axon-synthesis package

### PR DESCRIPTION
**Pull request by adrien-berchet**

### CODE REVIEW

**Comment by adrien-berchet** [comment]
Hi @jdcourcol 
I don't know receives notifications for this repo but here the PR for axon-synthesis (with a new banner! :star_struck: )

File: README.md [hunk 1]
```diff
@@ -46,6 +47,15 @@ Useful links:
...
 
+## Axon-Synthesis
+<img alt="Axon-Synthesis Banner" src="https://github.com/BlueBrain/axon-synthesis/raw/main/docs/source/BBP-Axon-Synthesis.jpg" width="600"/>
+
+Useful links:
+[GitHub repo](https://github.com/BlueBrain/axon-synthesis),
+[Documentation](https://axon-synthesis.readthedocs.io/).
+
+**Long-range axon synthesis in brain atlases.**
+
```
Comment (jdcourcol) [inline README.md:58]: may be add a bit of text after the title describing your repo

**Review by jdcourcol — CHANGES_REQUESTED** [review]

**Review by jdcourcol — APPROVED** [review]
nice banner. go ahead

File: README.md (hunk 1 above)
Comment (adrien-berchet) [reply README.md:58]: Done

**Comment by adrien-berchet** [comment]
Thanks!

**Pull request merged**